libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss

```diff
@@ -0,0 +1,63 @@
+/* Default styles for larger screens */
+.image-text-banner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  background: #dbc2f5 0% 0% no-repeat padding-box;
+}
+
+.image-container {
+  top: 1194px;
+  left: 240px;
+  width: 700px;
+  height: 750px;
+  flex: 1;
+  padding: 20px;
+  background: transparent
+    url('https://tellyspotting.kera.org/wp-content/uploads/2015/09/Mr.-Bean-at-Buckingham-Palace-for-his-25th-birthday.jpg')
+    0% 0% no-repeat padding-box;
+  margin-left: 100px;
+  border-radius: 50%; /* Create a vertical oval shape */
+}
+
+.text-container {
+  flex: 1;
+  padding: 20px;
+  width: 300px;
+  height: 200px;
+  text-align: left;
+  font: normal normal normal 32px/42px DM Sans;
```

Comment:
Also for fonts, use CSS variables specified in elements/theming/_variables.scss

Reply:
@AnnaL001 *resolved